docs/concept/Folder-Structure.md

```diff
@@ -57,19 +57,19 @@ app
 
 This is where your components will live, the reusable building blocks to create your screens. A handful of built-in components come with Ignite that are adaptable to any custom design system you wish to implement. Below are links to further documentation about each component:
```

Comment:
this Folder Structure page might not be necessary, a lot of the info here is duplicated in Boilerplate.md - so we probably don't want to maintain both